Sources/ResilientDecoding/ErrorReporting.swift

```diff
@@ -1,43 +1,114 @@
-// Created by George Leontiev on 3/28/20.
+// Created by George Leontiev on 3/25/20.
```

Comment:
Is this still true? Based on the PR description it seems like this may have changed.
![Screen Shot 2020-04-22 at 6 31 50 PM](https://user-images.githubusercontent.com/1791049/80049406-a7fdae00-84c7-11ea-8fd3-d064f83a22aa.png)

Reply:
It seems like this may still be true